### PR TITLE
Recipe for publishing to Azure DevOps wiki.

### PR DESCRIPTION
**Pull request by vitaly-pevgonen**
Adding recipe for publishing to Azure DevOps wiki.

### CODE REVIEW

File: docs/azure-devops-wiki.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Setup a Foam workspace
+
+Generate a Foam workspace using the [foam-template project](https://github.com/foambubble/foam-template). Change the remote to git repository in Azure DevOps, or copy all the files into a new Azure DevOps git repository.
+
+Define which document will be the wiki default page. To do that, create file with name `.order` in workspace root folder, with first line being document filename without `.md` extension. For project created from template, file would look like this:
+
+```
+readme
+```
+Push repository to remote in Azure DevOps.
```
Comment (jsjoeio) [inline docs/azure-devops-wiki.md:18]:
```suggestion
## Setup a Foam workspace

1. Generate a Foam workspace using the [foam-template project](https://github.com/foambubble/foam-template). 
2. Change the remote to a git repository in Azure DevOps, or copy all the files into a new Azure DevOps git repository.
3. Define which document will be the wiki default page. To do that, create a file called `.order` in the Foam workspace root folder, with first line being the document filename without `.md` extension. For a project created from the Foam template, the file would look like this:
```
readme
```
4. Push the repository to remote in Azure DevOps.
```

File: docs/azure-devops-wiki.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Publish repository to a wiki
+
+Process is better described in [Azure DevOps documentation](https://docs.microsoft.com/en-us/azure/devops/project/wiki/publish-repo-to-wiki). Navigate to your Azure DevOps project in web browser, choose **Overview** > **Wiki**. If you don't have wikis for your project, choose **Publish code as a wiki** on welcome page. Choose repository with your Foam workspace, branch (usually `master`), folder (for workspace created from foam-template it is `/`), and wiki name, and press **Publish**.
+
+Published workspace would look like this
+
+![Azure DevOps wiki](assets/images/azure-devops-wiki-demo.png)
+
+There is default TOC pane to the left of the wiki content. Here are listed all directories that present in Foam workspace, and all wiki pages. Pages names are derived from files names, and they are listed in alphabetical order. You may reorder pages by adding filenames without `.md` extension to `.order` file. Note that first entry in `.order` file defines wiki's home page.
```
Comment (jsjoeio) [inline docs/azure-devops-wiki.md:28]:
```suggestion

1. Navigate to your Azure DevOps project in a web browser.
2. Choose **Overview** > **Wiki**. If you don't have wikis for your project, choose **Publish code as a wiki** on welcome page. 
3. Choose repository with your Foam workspace, branch (usually `master` or `main`), folder (for workspace created from foam-template it is `/`), and wiki name, and press **Publish**.

A published workspace looks like this:

![Azure DevOps wiki](assets/images/azure-devops-wiki-demo.png)

There is default table of contents pane to the left of the wiki content. Here, you'll find a list of all directories that are present in your Foam workspace, and all wiki pages. Page names are derived from files names, and they are listed in alphabetical order. You may reorder pages by adding filenames without `.md` extension to `.order` file. 

_Note that first entry in `.order` file defines wiki's home page._

For more information, read the [Azure DevOps documentation](https://docs.microsoft.com/en-us/azure/devops/project/wiki/publish-repo-to-wiki). 
```

File: docs/azure-devops-wiki.md (hunk 1 above)
Comment (vitaly-pevgonen) [reply docs/azure-devops-wiki.md]: (GitHub UI somehow isn't happy about fenced code block here, will commit separately)

**Comment by vitaly-pevgonen** [comment]
Thank you a lot @jsjoeio for your edits, they definitely made this document much better. I also changed "wiki default page" to "wiki home page" in one place, so it is referred uniformly as home page through all the document.

**Comment by vitaly-pevgonen** [comment]
@jevakallio maybe all-contributors? 😳

**Comment by jsjoeio** [comment]
@all-contributors please add @vitaly-pevgonen for docs

**Pull request merged**